dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/DefaultTrackerObjectsDeletionService.java

```diff
@@ -113,7 +114,8 @@ public TrackerTypeReport deleteEnrollments(List<String> enrollments) throws NotF
       TrackedEntity te = enrollment.getTrackedEntity();
       te.setLastUpdatedByUserInfo(userInfoSnapshot);
 
-      apiEnrollmentService.deleteEnrollment(enrollment);
+      enrollmentStore.deleteEnrollment(enrollment);
```

Comment:
You could also use `manager.delete` here. In the end it will call the exact same method.